docs/contact-us.md

```diff
@@ -0,0 +1,9 @@
+# Contact Us
+
+Please reach out if there's any way that we can improve Token.js!
+
+Here are a few ways you can reach us:
+* [Discord](TODO)
+* [Schedule a meeting](https://calendly.com/sam_goldman/tokenjs)
+* Call or text: [+1 (516) 206-6928](tel:+15162066928)
+* Email: [sam@glade.so](mailto:sam@glade.so)
```

Comment:
We should also include this in the Github readme so it shows up there and on NPM.